packages/modules/common/configurable_monitoring.py

```diff
@@ -0,0 +1,22 @@
+import logging
+from typing import Callable
+
+
+log = logging.getLogger(__name__)
+
+
+class ConfigurableMonitoring():
+    def __init__(self,
+                 start_initializer: Callable[[], None],
+                 stop_initializer: Callable[[], None]) -> None:
+        try:
+            self._start_monitoring = start_initializer()
+            self._stop_monitoring = stop_initializer()
```

Comment:
Sollen die Funktionen hier ausgeführt oder zugewiesen werden?
```suggestion
            self._start_monitoring = start_initializer
            self._stop_monitoring = stop_initializer
```